libsrc/gamate/vga.inc

```diff
@@ -0,0 +1,228 @@
+
+; VGA charset for the PC-Engine conio implementation
+
+        .byte   $00, $00, $00, $00, $00, $00, $00, $00
+        ; 1
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %11111111
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        ; 2
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        ; 3
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %11111111
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        ; 4
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00011111
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        ; 5
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %11110000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        ; 6
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00011111
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        ; 7
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %11110000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        ; 8
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %11111111
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        ; 9
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %11111111
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        ; 10 (LF)
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        .byte   %00000000
+        ; 11
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %11110000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        ; 12
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00011111
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+        .byte   %00010000
+
+        .byte   $3F, $33, $3F, $30, $30, $70, $F0, $E0
```

Comment:
nice... sure. perhaps. but it wont be me who is putting a lot of effort into it when i dont need it :) (IMHO the proper solution to all the charsets that are in various parts of the lib is having them in eg .png format plus some kind of converter program - which then could be used by users to replace the default chars too)